views/athlete.pug

```diff
@@ -2,26 +2,73 @@ extends layout
 
 
 mixin table(header,data, type)
-	- var type = type || ""
-	table.table(class=type)
-		thead
-			tr(align="center")
-				each head, i in header
-					th #{head}
-		tbody
-			each row,j in data
-				tr
-					each col,k in row
-						td(align="center") #{col}
-	
+    - var type = type || ""
+    table.table(class=type)
+        thead
+            tr(align="center")
+                each head, i in header
+                    th #{head}
+        tbody
+            each row,j in data
+                tr
+                    each col,k in row
+                        td(id='max'+k, align="center", contenteditable='true') #{col} &nbsp
+                             i.far.fa-edit
+    
 mixin table-condensed(header,data)
-	+table(header,data,"table-condensed")
+    +table(header,data,"table-condensed")
+append scripts
+    script(src="https://d3js.org/d3.v3.min.js")
+    script(src='/js/athleteViewHistoryCtrl.js')
+    style(type='text/css').
+        .line {
```

Comment:
Could you move these styles into the SASS file?

Reply:
I put them here so they do not conflict with the createPhase styles